docs/content/tutorials/kic/deploy-with-kic.md

```diff
@@ -465,3 +473,88 @@ Then you can navigate your browser to `localhost:3000` to view Grafana.
 Here is a view of the provided "NGINX Mesh Top" dashboard:
 
 {{< img src="/img/grafana.png" >}}
+
+## Integrating NGINX Ingress Controller with NSM
```

Comment:
NSM shouldn't be used in our public docs. Be sure to use the full NGINX Service Mesh phrasing (and be sure to update the link to this section.